### ISSUE TITLE
Link to deprecated manifest v2 docs in extension publishing guide

### ISSUE THREAD

**Issue by MiWeiss** [GoogleChrome/developer.chrome.com#931]
**Describe the bug**
There's still a link to the deprecated extension manifest v2 docs in the "Publish in the Chrome Web Store" guide. 

**To Reproduce**
1. Open the docs [here](https://developer.chrome.com/docs/webstore/publish/#create-your-items-zip-file)
2. The link `files for your extension` points to https://developer.chrome.com/docs/extensions/mv2/overview#files
3. Clicking the link shows 

> Warning
> The page you're viewing describes extensions using Manifest V2. Now that Manifest V3 has launched, we strongly recommend that you use it for any new extensions that you create.

**Screenshots**
![image](https://user-images.githubusercontent.com/4815944/123558098-deeae600-d794-11eb-92c8-9b55b08536db.png)

**Comment by robdodson** [comment]
@dotproto should this be updated to mv3?

**Comment by dotproto** [comment]
@robdodson, yeah, it should be. This is a quick fix and I just wrapped up some other work, so I'll make a PR for this now.
